plugins/wallabag.koplugin/main.lua

```diff
@@ -981,6 +993,7 @@ function Wallabag:saveSettings()
         is_sync_remote_delete = self.is_sync_remote_delete,
         articles_per_sync     = self.articles_per_sync,
         remove_finished_from_history = self.remove_finished_from_history,
+        download_queue      = self.download_queue,
```

Comment:
If you do ASCII art you have to do it properly. ;-)
```suggestion
        download_queue        = self.download_queue,
```